fmriprep/workflows/bold/confounds.py

```diff
@@ -428,40 +427,53 @@ def init_ica_aroma_wf(template, metadata, mem_gb, omp_nthreads,
 
     **Parameters**
 
+        aroma_melodic_dim: int or None
+            Set the dimensionality of the Melodic ICA decomposition
+            If None, MELODIC automatically estimates dimensionality.
+        ignore_aroma_err : bool
+            Do not fail on ICA-AROMA errors
+        mem_gb : float
+            Size of BOLD file in GB
+        metadata : dict
+            BIDS metadata for BOLD file
+        name : str
+            Name of workflow (default: ``bold_mni_trans_wf``)
+        omp_nthreads : int
+            Maximum number of threads an individual process may use
         template : str
             Spatial normalization template used as target when that
             registration step was previously calculated with
             :py:func:`~fmriprep.workflows.bold.registration.init_bold_reg_wf`.
             The template must be one of the MNI templates (fMRIPrep uses
             ``MNI152NLin2009cAsym`` by default).
-        metadata : dict
-            BIDS metadata for BOLD file
-        mem_gb : float
-            Size of BOLD file in GB
-        omp_nthreads : int
-            Maximum number of threads an individual process may use
-        name : str
-            Name of workflow (default: ``bold_mni_trans_wf``)
         susan_fwhm : float
             Kernel width (FWHM in mm) for the smoothing step with
             FSL ``susan`` (default: 6.0mm)
         use_fieldwarp : bool
             Include SDC warp in single-shot transform from BOLD to MNI
-        ignore_aroma_err : bool
-            Do not fail on ICA-AROMA errors
-        aroma_melodic_dim: int or None
-            Set the dimensionality of the Melodic ICA decomposition
-            If None, MELODIC automatically estimates dimensionality.
```

Comment:
It looks like you sorted these. I would keep these in the order that they are in the parameter list. If we want to reorder that, we can do that in another PR, but that's an API change, and we should avoid it unless there's a good reason.